### ISSUE TITLE
COAM\Penalty - Imposed penalty External link to EAB

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-coam#79]
#### Description:

An operator can click on a link that takes them to the EAB website if they want to appeal their imposed penalty.

#### Acceptance Criteria:

As an operator, given that I've been issued an imposed penalty, when I feel this has been done incorrectly or unfairly, then I can read information about how to appeal my penalty and click on a link that takes me to the [Environmental Appeal Board](https://www.bceab.ca).

- [ ] Include text about how an imposed penalty can be appealed and a link to the EAB on the penalty invoice
- [ ] Link to EAB: https://www.bceab.ca 

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

#### Definition of Ready (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

#### Definition of Done (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By/Blocking
Blocked by #77 

#### Notes:

-
